src/main/java/frc/robot/subsystems/vision/LimelightLogInputs.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.subsystems.vision;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class LimelightLogInputs implements LoggableInputs {
```

Comment:
You should use the `@AutoLog` annotation which was introduced in the 2023 version of AdvantageKit. Read more [here](https://github.com/Mechanical-Advantage/AdvantageKit/blob/f80d5531d7e44618f622c4079bb38032053b13a4/docs/NEW-FOR-2023.md#autolog-annotation).

Reply:
You didn't fully utilize the annotation. Please read the documentation again. I'm here to help ☺️.